Python/pywarpx/Bucket.py

```diff
@@ -59,7 +59,9 @@ def attrlist(self):
         for attr, value in self.argvattrs.items():
             if value is None:
                 continue
-            # --- repr is applied to value so that for floats, all of the digits are included.
+            # --- str (and repr) crop some digits for floats. TODO: we should format
```

Comment:
Doing some quick tests show that both repr and str will include all digits, as wil f-strings.
```
>>> repr(np.pi)
'3.141592653589793'
>>> str(np.pi)
'3.141592653589793'
>>> f'{np.pi}'
'3.141592653589793'
>>> np.pi - float(f'{np.pi}')
0.0
>>> 
```
So, maybe use the f-string to avoid the explicit call to `str`?